01_04_Damage_Types_and_Armor_Types.txt

```diff
@@ -22,10 +22,10 @@ ballistic armor is hit with a laser, they receive damage as if they weren't
 wearing any armor.
 
 Explosives:
-    Certain grenades and devices have Explosive (concussive) damage. Explosive 
-Damage ignores armor and shields (explosive damage is essentially, a crushing 
-force). Fragmentation grenades are considered ballistic weapons, as their 
-primary damage delivery method is shrapnel. 
+    Certain grenades and devices have Explosive damage, which ignores armor and 
```

Comment:
Scope creep: Do we want to standardize to weapons "having" or "dealing" damage types? I personally prefer "dealing".

Reply:
Agree